org.gnome.gnome-mahjongg.yaml

```diff
@@ -0,0 +1,28 @@
+app-id: org.gnome.gnome-mahjongg
```

Comment:
the app-id could be just `org.gnome.mahjongg`

Reply:
I see that you have opened an issue upstream about that https://gitlab.gnome.org/GNOME/gnome-mahjongg/issues/15
You could use that already. Will send a MR to fix that upstream too

Reply:
It could and should indeed. But the app registers itself under the name gnome-mahjongg and when I tried to name it org.gnome.Mahjongg it didn't work (issues with the name of the dbus service).
It requires changes in the app itself, I filed a bug about it upstream: https://gitlab.gnome.org/GNOME/gnome-mahjongg/issues/15
Until it gets resolved, the app id just has to be org.gnome.gnome-mahjongg.

Reply:
https://gitlab.gnome.org/GNOME/gnome-mahjongg/merge_requests/5

Reply:
https://gitlab.gnome.org/GNOME/gnome-mahjongg/merge_requests/7

Reply:
That's the master. I'm building the flatpak from stable. Would that PR be portable to the stable version? I could also build from master, it wouldn't only solve this issue, but I could also use meson instead of autotools.

Reply:
yeah; you can build from stable. But you will need to change the app-id later :/ so better wait until it's fixed upstream first

Reply:
Could one not just add the changes as a patch for now ? So that it is stable + accepted patch to have the fixed app-id ? Or is that not acceptable for flathub ?

Reply:
Ah i see the merge hasn't been approved upstream yet, ignore me :-)